backend/app/controllers/backend/video_collections_controller.rb

```diff
@@ -29,7 +29,7 @@ def update
     end
 
     def new
-      @videos = Video.where(album_id: id).order(:title)
+      @videos = Video.order(:title)
```

Comment:
Probably worth getting the videos that don't belong to any VideoCollection. to avoid adding the same video to multiple albums.